This fixes architecture mismatch, when building
on Cygwin 64bit and in 32bit mode
diff -ru nss.orig/nss/Makefile nss/nss/Makefile
--- a/nss.orig/nss/Makefile       2016-02-26 01:00:52.146713100 +0100
+++ b/nss/nss/Makefile    2016-02-26 01:02:05.303560100 +0100
@@ -59,6 +59,8 @@
 # Translate coreconf build options to NSPR configure options.
 #

+NSPR_CONFIGURE_OPTS += --host=i686-pc-cygwin
+
 ifeq ($(OS_TARGET),Android)
 NSPR_CONFIGURE_OPTS += --with-android-ndk=$(ANDROID_NDK) \
                        --target=$(ANDROID_PREFIX) \
